src/panels/lovelace/views/hui-view.ts

```diff
@@ -264,13 +261,9 @@ export class HUIView extends LitElement {
     const elements: HUIView["_badges"] = [];
     const badges = processConfigEntities(config.badges);
     for (const badge of badges) {
-      const element = document.createElement("ha-state-label-badge");
+      const element = createBadgeElement(badge);
       const entityId = badge.entity;
```

Comment:
We no longer need to store this. This was used because hui-view had knowledge on how to update `ha-state-label-badge` in the past.